### PR TITLE
(feat) auth redesign

### PR DESCRIPTION
**Pull request by benjaminstrasser**
- [x] redesign auth layout
- [x] redesign signup page
- [x] redesign signin page
- [x] created theme picker component

### CODE REVIEW

**Review by jjnp — DISMISSED** [review]

File: src/routes/(auth)/+layout.svelte [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<script lang="ts">
+	import Check from 'lucide-svelte/icons/check'
+
```
Comment (jjnp) [inline src/routes/(auth)/+layout.svelte:3]: whitespace could be removed, but that's a nit.

File: src/routes/(auth)/signup/schema.ts [hunk 2]
```diff
@@ -9,7 +9,8 @@ export const signupSchema = z
 			.string({ required_error: 'Password is required' })
 			.min(8, { message: 'Password must be at least 8 characters' })
 			.trim(),
-		confirmPassword: z.string().trim()
+		confirmPassword: z.string().trim(),
```
Comment (jjnp) [inline src/routes/(auth)/signup/schema.ts:12]: Why trim()?

File: src/routes/(auth)/signup/signup-form.svelte [hunk 3]
```diff
@@ -26,45 +26,69 @@
...
-			<Card.Title>Sign Up to tiny-tms</Card.Title>
-			<Card.Description>Register to tiny-tms and start doing you translations</Card.Description>
-		</Card.Header>
-		<Card.Content>
+<div class="flex items-center justify-center">
+	<div class="w-full max-w-2xl">
+		<div class="text-center">
+			<h2 class="mt-6 text-center text-3xl font-extrabold">Register</h2>
+			<p class="mt-2 text-center text-sm">
+				Enter you informationto create an user account for this Tiny-TMS instance
```
Comment (jjnp) [inline src/routes/(auth)/signup/signup-form.svelte:36]: Typo in `informationto`

**Review by jjnp — APPROVED** [review]

File: src/routes/(auth)/+layout.svelte (hunk 1 above)
Comment (benjaminstrasser) [reply src/routes/(auth)/+layout.svelte]: fixed

File: src/routes/(auth)/signup/signup-form.svelte (hunk 3 above)
Comment (benjaminstrasser) [reply src/routes/(auth)/signup/signup-form.svelte]: fixed

**Review by jjnp — APPROVED** [review]

**Pull request merged**